tests/Resources/Page/OCPDashboard/Monitoring/Metrics.robot

```diff
@@ -21,27 +21,34 @@ Verify Query Results Dont Contain Data
   Should Be True   not ${metrics_query_results_contain_data}
 
 Run Query
-  [Arguments]  ${query}  ${retry-attempts}=10
+  [Arguments]  ${query}  ${retry_attempts}=10
   Wait Until Page Contains Element    ${METRICS_QUERY_TEXTAREA}  timeout=30
   Input Text   ${METRICS_QUERY_TEXTAREA}  ${query}
 
-  ${end-range}=   Evaluate    ${retry-attempts} + 1
+  ${end-range}=   Evaluate    ${retry_attempts} + 1
```

Comment:
You could use  `${end-range} =    set variable   ${${retry_attempts} + 1}` so that you can avoid evaluate a python expression